torch/jit/_script.py

```diff
@@ -36,6 +36,14 @@
 from torch.overrides import (
     has_torch_function, has_torch_function_unary, has_torch_function_variadic)
 
+_IS_MONKEYTYPE_INSTALLED = True
```

Comment:
The import guard can be avoided if we do this:
https://github.com/pytorch/pytorch/pull/54513/files#r609120259

Reply:
Fixed now